test_exercise1.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+import unittest
+from exercise1 import print_greeting
+
+class GreetUserTest(unittest.TestCase):
+    """Test for function print_greeting"""
+
+    def test_greeting(self):
```

Comment:
So this kind of validates my comment above because the name of the test is `test_greeting`, not `test_print_greeting`. You'll want to make sure your test names match your function names. Per my recommendation above, this function should be called `generate_greeting`, or something similar.